src/settings/Settings.ts

```diff
@@ -19,6 +19,18 @@ export interface MediaDbPluginSettings {
 	openNoteInNewTab: boolean;
 	useDefaultFrontMatter: boolean;
 	enableTemplaterIntegration: boolean;
+	OMDbAPImovie: boolean;
+	OMDbAPIseries: boolean;
+	OMDbAPIgame: boolean;
+	MALAPImovie: boolean;
+	MALAPIseries: boolean;
+	SteamAPIgame: boolean;
+	MobyGamesAPIgame: boolean;
+	BoardGameGeekAPIdefault: boolean;
```

Comment:
what are the variables ending in `default` for?

Reply:
I added those to try and always enable certain APIs even if there was no media type but the way I went about it was wrong, this is now fixed by only checking if it's false that an API is turned off.